src/object.d

```diff
@@ -2309,6 +2310,145 @@ inout(V) get(K, V)(inout(V[K])* aa, K key, lazy inout(V) defaultValue)
     }
 }
 
+unittest
+{
+    static class T
+    {
+        static size_t count;
+        this() { ++count; }
+    }
+
+    T[string] aa;
+
+    auto a = new T;
+    aa["foo"] = a;
+    assert(T.count == 1);
+    auto b = aa.get("foo", new T);
+    assert(T.count == 1);
+    assert(b is a);
+    auto c = aa.get("bar", new T);
+    assert(T.count == 2);
+    assert(c !is a);
+
+    //Obviously get doesn't add.
+    assert("bar" !in aa);
+}
+
+ref V require(K, V)(ref V[K] aa, K key, lazy V value = V.init) pure
```

Comment:
new public symbols need a DDoc header with appropriate summary, `params`, and `returns`.  See https://dlang.org/spec/ddoc.html#standard_sections

Reply:
No problem, I can add that. There is some documentation added in dlang/dlang.org#2343 if you could review and make comments.

Reply:
The DDoc header is still required for library documentation at https://dlang.org/phobos/object.html

Reply:
>No problem, I can add that.